### PR TITLE
[draft] Announce renaming `master` to `main`

### PR DESCRIPTION
**Pull request by JonRowe**

### CODE REVIEW

File: source/blog/2020-06-16-rspec-branching-strategy.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+RSpec's source code is hosted on Github. By default git, and thus Github has
+historically used `master` as the name of the default, or main branch. This
+term has sat uneasily on the minds of the team and with the recent community
+led moves to eliminate this terminology from our technology stacks, we are going
+to follow suit and rename this branch.
+
+We currently favour `main` as the new name (it has shared muscle memory with
+the old) but we also recognise that having community consensus around a
+default name is important as it reduces the amount of surprise encountered by
+developers. So we wil wait for the community to settle on a new default name,
```
Comment (benoittgt) [inline source/blog/2020-06-16-rspec-branching-strategy.md:28]:
```suggestion
 developers. So we will wait for the community to settle on a new default name,
```

File: source/blog/2020-06-16-rspec-branching-strategy.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+We have always encouraged the use of our main branch for getting pre-release
+features before we release them as a group of gems, this means there are likely
+people using the `master` branch via their Gemfiles. We will deprecate
+`master` by means of notice in the documentation, and possibly a post install
+/ on usage message from RSpec itself warning of the fact the branch will
+become stale and not be updated further. Additionally we will look at
+using branch protection rules preventing PRs against `master` from that date.
+
+## Steps we intend to take.
```
Comment (benoittgt) [inline source/blog/2020-06-16-rspec-branching-strategy.md:50]: Maybe be more "guiding". Like "Steps we will do"

File: source/blog/2020-06-16-rspec-branching-strategy.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+title: RSpec Branching Strategy
+author: Jon Rowe
+---
+
+# The short version.
+
+RSpec repositories will deprecate the use of `master` for the default branch
+name. We will wait for a consensus on the new name for the default branch,
+in particular what new default is choosen by Github.
```
Comment (xaviershay) [inline source/blog/2020-06-16-rspec-branching-strategy.md:10]:
```suggestion
in particular what new default is chosen by Github.
```

File: source/blog/2020-06-16-rspec-branching-strategy.md (hunk 2 above)
Comment (JonRowe) [reply source/blog/2020-06-16-rspec-branching-strategy.md:50]: I'm happier with this as it is, it reads better in my opinion.

File: source/blog/2020-06-16-rspec-branching-strategy.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+---
+title: RSpec Branching Strategy
+author: Jon Rowe
+---
+
+# The short version.
```
Comment (pirj) [inline source/blog/2020-06-16-rspec-branching-strategy.md:6]: Maybe remove the trailing periods from titles?

File: source/blog/2020-06-16-rspec-branching-strategy.md (hunk 2 above)
Comment (benoittgt) [reply source/blog/2020-06-16-rspec-branching-strategy.md:50]: No problem :)

File: source/blog/2020-06-16-rspec-branching-strategy.md (hunk 4 above)
Comment (JonRowe) [reply source/blog/2020-06-16-rspec-branching-strategy.md]: I agree looks better

**Comment by benoittgt** [comment]
Thanks!

**Comment by JonRowe** [comment]
Checked on staging and publishing now

**Pull request merged**